data/en/rsi_transformation.json

```diff
@@ -0,0 +1,576 @@
+                        "id": "reporting-period-question",
+                        "title": "What are the dates of the period that you will be reporting for?",
+                        "period_limits": {
+                            "minimum": {
+                                "days": 23
+                            },
+                            "maximum": {
+                                "days": 50
+                            }
+                        },
+                        "answers": [{
+                                "type": "Date",
+                                "id": "period-from",
+                                "label": "From",
+                                "mandatory": true,
+                                "q_code": "11",
+                                "minimum": {
+                                    "meta": "ref_p_start_date",
+                                    "offset_by": {
+                                        "days": -19
+                                    }
+                                }
+                            },
+                            {
+                                "type": "Date",
+                                "id": "period-to",
+                                "label": "To",
+                                "mandatory": true,
+                                "q_code": "12",
+                                "maximum": {
+                                    "meta": "ref_p_end_date",
+                                    "offset_by": {
+                                        "days": 20
+                                    }
+                                }
+                            }
+                        ]
+                    }]
+                },
+                {
+                    "type": "Question",
+                    "id": "total-turnover-block",
+                    "questions": [{
+                        "guidance": {
+                            "content": [{
+                                    "title": "Include",
+                                    "list": [
+                                        "commission and fees",
+                                        "exports"
+                                    ]
+                                },
+                                {
+                                    "title": "Exclude",
+                                    "list": [
+                                        "VAT",
+                                        "excise duties",
+                                        "grants and subsidies"
+                                    ]
+                                }
+                            ]
+                        },
+                        "answers": [{
+                            "id": "total-turnover-answer",
+                            "label": "Total turnover excluding VAT",
+                            "mandatory": false,
+                            "q_code": "20a",
```

Comment:
Just want to check if this is 0 we route past next question and go significant changes? as its not on the spec

Reply:
This goes to confirmation question confirm-zero-turnover-answer. That routes to significant changes.